src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -182,7 +188,7 @@ public override TypeKind TypeKind
         {
             get
             {
-                return TypeKind.Tuple;
+                return _underlyingType.TypeKind;
```

Comment:
I think it can only be struct or class. Requirement to have fields rules out enums, delegates and the like.
Will change to be a struct if underlying is a struct, class otherwise.